doc/rtd/explanation/about-cloud-config.rst

```diff
@@ -0,0 +1,85 @@
+.. _about-cloud-config:
+
+About the cloud-config file
+***************************
+
+The ``#cloud-config`` file is a type of user data that cloud-init can consume
+to automatically set up various aspects of the system.
+
+It overrides all other types of user data, so if there is a conflict between
+the config you specify in other :ref:`user data formats <user_data_formats>`,
+the contents of the ``#cloud-config`` file will take precendence.
+
+Note that networks are not configurable via the ``#cloud-config`` file because
+:ref:`network configuration <network_config>` occurs before user data is
+consumed and applied.
+
+How do I create a cloud-config file?
+====================================
+
+The cloud-config file can be written in any valid YAML but the first line
+**must start** with ``#cloud-config``. This line identifies the file to
+cloud-init and ensures that it will be processed as intended.
+
+After the first line, every aspect of the system's configuration is controlled
+through specific cloud-init **modules**. Each module included in the
+``cloud-config`` file can be thought of as a section.
+
+Let us consider the example of the "keyboard" module. The module has a
+corresponding top-level key (``keyboard:``, in this case), and beneath that
+the various configuration parameters are defined:
+
+.. code-block:: yaml
+
+   keyboard:
```

Comment:
Lets add the `#cloud-config` header here.

Reply:
Good catch!